libs/@local/hash-graph-types/LICENSE.md

```diff
@@ -0,0 +1,3 @@
+# License
```

Comment:
@TimDiekmann Shouldn't this be AGPL?

Reply:
I was assuming as we want it at the awesome-blocks page and this is only a small library that "only" transforms some JSON schemas we were about to publicize it under MIT/Apache, especially as the [type-system is also MIT/Apache](https://github.com/blockprotocol/blockprotocol/blob/main/libs/%40blockprotocol/type-system/crate/LICENSE.md). Still, I was about to check with you as soon as the PR is ready for review.

Reply:
You want me to change it to AGPL?

Reply:
Let's make this AGPL for now, as it's only at the point of being `@local` anyway (even if we are planning to link from `awesome-blocks`). We can can examine this in more depth with a view to liberalizing it later. Thanks!